Formula/wiremock-standalone.rb

```diff
@@ -0,0 +1,19 @@
+class WiremockStandalone < Formula
+  desc "Simulator for HTTP-based APIs"
+  homepage "http://wiremock.org"
+  url "https://search.maven.org/remotecontent?filepath=com/github/tomakehurst/wiremock-standalone/2.12.0/wiremock-standalone-2.12.0.jar"
```

Comment:
Looks like there's a new version now.

Reply:
@DomT4 I updated the version to the most recent one at the time of this writing.